h/exceptions.py

```diff
@@ -1,52 +1,46 @@
 # -*- coding: utf-8 -*-
-
 """Exceptions raised by the h application."""
 
 from __future__ import unicode_literals
 
-from h.i18n import TranslationString as _  # noqa: N813
+from pyramid import httpexceptions
 
+from h.i18n import TranslationString as _  # noqa: N813
 
-class APIError(Exception):
 
-    """Base exception for problems handling API requests."""
+class APIError(httpexceptions.HTTPError):
+    """A problem handling an API request."""
 
-    def __init__(self, message, status_code=500):
-        self.status_code = status_code
-        super(APIError, self).__init__(message)
+    def __init__(self, detail, status_code=500):
+        super(APIError, self).__init__(detail, status_code=status_code)
```

Comment:
`message` is renamed to `detail` throughout because that's what Pyramid's `HTTPError`, which these are now subclasses of, calls it.